gma/src/android/native_ad_image_android.cc

```diff
@@ -71,6 +72,10 @@ NativeAdImage::NativeAdImage(
   FIREBASE_ASSERT(j_uri);
   internal_->uri = util::JniUriToString(env, j_uri);
 
+  // Images requested with an android user agent may return webp images. Trim
+  // webp parameter from image url to get the original JPG/PNG image.
+  internal_->uri = std::regex_replace(internal_->uri, std::regex("-rw"), "");
```

Comment:
Is there a reason to use a regex for something that's just a text replacement?

Reply:
No particular reason. regex_replace syntax seemed easier to read and performance is similar to a regular replace. 
Would you prefer something like this:  internal_->uri.replace(internal_->uri.find("-rw"),3,"");

Reply:
Yes, I'd prefer that - regex is a lot of overhead for just a text replacment.

(Also, if you do it the other way, definitely also check that "-rw" is in the string first to avoid errors.)

Reply:
That makes sense. Updated it in the latest commit. Thanks for the review!